src/hgraph/_operators/_operators.py

```diff
@@ -628,50 +628,39 @@ def my_getattr(ts: TS[MyType], attr: str) -> TS[SomeScalar]:
 
 
 @operator
-def min_(lhs: TIME_SERIES_TYPE,
-         rhs: TIME_SERIES_TYPE = None,
-         default_value: TIME_SERIES_TYPE = None
-         ) -> TIME_SERIES_TYPE:
+def min_(*ts: TSL[TS[SCALAR], SIZE], default_value: TIME_SERIES_TYPE = None) -> TIME_SERIES_TYPE:
     """
     This represents the `min` operator for time series types.
 
     Unary implies the min over the latest TS value for collection types, or running min for non-collection types
-    Binary implies the min between the lhs and rhs
-    Multi arg implies the min over all the arguments
-
-    TODO - implement multi-arg signature
+    Binary or multi arg implies item-wise min over all the arguments for collection types,
+    or the minimum scalar value for scalar types
     """
-    raise WiringError(f"operator min_() is not implemented for {lhs.output_type} and {rhs.output_type}")
+    raise WiringError(f"operator min_() is not implemented for {ts.output_type}")
 
 
 @operator
-def max_(lhs: TIME_SERIES_TYPE,
-         rhs: TIME_SERIES_TYPE = None,
-         default_value: TIME_SERIES_TYPE = None) -> TIME_SERIES_TYPE:
+def max_(*ts: TSL[TS[SCALAR], SIZE], default_value: TIME_SERIES_TYPE = None) -> TIME_SERIES_TYPE:
     """
     The `max` operator for time series types.
 
     Unary implies the max over the latest TS value for collection types, or running max for non-collection types
-    Binary implies the max between the lhs and rhs
-    Multi arg implies the max over all the arguments
-
-    TODO - implement multi-arg signature
+    Binary or multi arg implies item-wise max over all the arguments for collection types,
+    or the maximum scalar value for scalar types
     """
-    raise WiringError(f"operator max_() is not implemented for {lhs.output_type} and {rhs.output_type}")
+    raise WiringError(f"operator max_() is not implemented for {ts.output_type}")
 
 
 @operator
-def sum_(lhs: TIME_SERIES_TYPE, rhs: TIME_SERIES_TYPE = None) -> TIME_SERIES_TYPE_2:
+def sum_(*ts: TSL[TS[SCALAR], SIZE], default_value: TIME_SERIES_TYPE = None) -> TIME_SERIES_TYPE:
```

Comment:
If we define the ts to be TSL[TS[SCALAR]], ...] then surely the default should be TS[SCALAR] and for output? For the binary operator it is possible to have different lhs and rhs types -> lhs type. But in the general case as defined here?